passport.go

```diff
@@ -130,20 +130,16 @@ func (v *Verifier) validateBase(zkProof zkptypes.ZKProof) error {
 }
 
 func (v *Verifier) validateBirthDate(signals []string) val.Errors {
-	allowedBirthDate := time.Now().UTC().AddDate(-v.opts.age, 0, 0)
-
-	return val.Errors{
-		"pub_signals/birth_date": val.Validate(signals[BirthDate], val.When(
-			!isEmptyZKDate(signals[BirthDate]) || v.opts.age != -1,
-			val.Required,
-			beforeDate(allowedBirthDate),
-		)),
-		"pub_signals/birth_date_upper_bound": val.Validate(signals[BirthdateUpperBound], val.When(
-			!isEmptyZKDate(signals[BirthdateUpperBound]) || v.opts.age != -1,
-			val.Required,
-			equalDate(allowedBirthDate),
-		)),
+	if v.opts.age == -1 {
+		return val.Errors{}
```

Comment:
```suggestion
		return nil
```
We can do it everywhere, because it's an alias for map